main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"crypto/md5"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/spf13/cobra"
+)
+
+var (
+	randSource = rand.Reader
+	filename   string
+)
+
+func main() {
+	cmdGenerateKey := &cobra.Command{
+		Use:   "generate",
+		Short: "generate client id and client secret",
+		Long:  "generate certificate, allow server to issue sdk token to use with BitmarkSDK",
+		Run:   generateClientKeys,
+		Args:  cobra.NoArgs,
+	}
+	cmdGenerateKey.Flags().StringVarP(&filename, "file", "f", "key.pem", "client certificate file")
+
+	cmdIssueSDKToken := &cobra.Command{
+		Use:   "issue [account]",
+		Short: "issue new jwt token using generated pem file with an account",
+		Run:   issueSDKToken,
+		Args:  cobra.ExactArgs(1),
+	}
+	cmdIssueSDKToken.Flags().StringVarP(&filename, "file", "f", "key.pem", "client certificate file")
+
+	var rootCmd = &cobra.Command{}
+	rootCmd.AddCommand(cmdGenerateKey, cmdIssueSDKToken)
+	rootCmd.Execute()
+}
+
+func generateClientKeys(cmd *cobra.Command, args []string) {
+	key, err := rsa.GenerateKey(randSource, 2048)
+	checkError(err)
+
+	outFile, err := os.Create(filename)
+	checkError(err)
+	defer outFile.Close()
+
+	var privateKey = &pem.Block{
+		Type:  "PRIVATE KEY",
+		Bytes: x509.MarshalPKCS1PrivateKey(key),
+	}
+
+	err = pem.Encode(outFile, privateKey)
+	checkError(err)
+
+	pubkey := x509.MarshalPKCS1PublicKey(&key.PublicKey)
+	pubkeyBase64 := base64.StdEncoding.EncodeToString(pubkey)
+	fmt.Println("Client ID: ", pubkeyBase64)
+	fmt.Println("Saved client secret to: " + filename)
+}
+
+func issueSDKToken(cmd *cobra.Command, args []string) {
```

Comment:
Can issue token by the library you wrote?

Reply:
Yes, I intended to change to use the module for this one after it being merged to master.